src/routes/about/organization/[officePage]/+page.server.ts

```diff
@@ -47,37 +45,47 @@ const query = gql`
             }
           }
         }
-        metadata {
+        pageMetadata {
           ... on PageMetadata {
+            sys {
+              id
+            }
             slug
-            metaTitle
-            metaDescription
           }
         }
       }
     }
   }
 `;
 
-export const load = (async ({ params }): Promise<OfficePage> => {
-  const { slug } = params;
+export const load = async ({ parent, params }) => {
+  const { pageMetadataMap } = await parent();
+  const slug = params.officePage;
   if (CONTENTFUL_SPACE_ID && CONTENTFUL_DELIVERY_API_TOKEN) {
     const client = getContentfulClient({
       spaceID: CONTENTFUL_SPACE_ID,
       token: CONTENTFUL_DELIVERY_API_TOKEN,
     });
     const data = await client.fetch<OfficePageQuery>(printQuery(query));
     if (data) {
-      const officePages = data?.officePageCollection?.items as OfficePage[];
-      const matchedOfficePage = officePages.find(
-        (officePage) => officePage.metadata?.slug === slug
+      const matchedOfficePage = data?.officePageCollection?.items?.find(
+        (officePage) => officePage?.pageMetadata?.slug === slug
       );
       if (matchedOfficePage) {
-        return matchedOfficePage;
+        const pageMetadataId = matchedOfficePage?.pageMetadata?.sys?.id;
+        if (pageMetadataId) {
+          const pageMetadata = pageMetadataMap.get(pageMetadataId);
+          if (pageMetadata) {
+            return {
+              officePage: matchedOfficePage,
+              pageMetadata,
+            };
+          }
+        }
       }
     }
   } else {
-    return OfficePageTestContent;
+    return { officePage: OfficePageTestContent, pageMetadata: {} };
   }
   throw error(404);
-}) satisfies PageServerLoad;
```

Comment:
I removed this since it didn't seem necessary, but I can re-add it (along with adding `PageServerLoad` to other `+page.server.ts` files and `LayoutServerLoad` to `+layout.server.ts` files) if y'all think we should have it.